app/src/main/java/ch/epfl/cs311/wanderwave/model/data/TrackRecord.kt

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.cs311.wanderwave.model.data
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "track_records")
+data class TrackRecord(
+    @PrimaryKey(autoGenerate = true) val id: Int = 0, // Unique ID for each record
```

Comment:
I guess the primary key could also directly be the timestamp since it is unique (one user won't add more than one track in one millisecond). But a separate ID is also fine, just a suggestion.